packages/framework-integration-tests/src/commands/MoveStock.ts

```diff
@@ -16,22 +16,26 @@ export class MoveStock {
     readonly quantity: number
   ) {}
 
-  public async handle(register: Register): Promise<void> {
-    const stock = await Booster.fetchEntitySnapshot(Stock, this.productID)
-    if (this.enoughStock(stock)) {
-      register.events(new StockMoved(this.productID, this.origin, this.destination, this.quantity))
+  public static async handle(command: MoveStock, register: Register): Promise<void> {
+    const stock = await Booster.fetchEntitySnapshot(Stock, command.productID)
+    if (MoveStock.enoughStock(command, stock)) {
```

Comment:
I think it is ok (and easier to read( to have the method defined in the instance of the command, as you did in the documentation.

Reply:
Fixed https://github.com/boostercloud/booster/pull/333/commits/a2c62c156f2da88ede722320c3a6b942bfa0685d thanks